Dockerfile.template

```diff
@@ -74,7 +74,9 @@ RUN set -ex; \
 		gpgconf --kill all; \
 	else \
 		:; \
-	fi
+	fi ; \
+	apt-get remove -y dirmngr gpg-agent ; \
```

Comment:
Removing packages in a later `RUN` line means that it will still take space in the image. We instead recommend adding and removing packages in the same `RUN` that they are used if they are not meant to be part of the final image.